features/security_status.feature

```diff
@@ -859,3 +859,105 @@ Feature: Security status command behavior
 
       Ubuntu Pro is not available for non-LTS releases\.
       """
+
+  Scenario Outline: Pass custom APT configuration to the Client for updates information
+    Given a `<release>` `<machine_type>` machine with ubuntu-advantage-tools installed
+    # Get the system up to date
+    When I apt update
+    And I apt upgrade
+    # Install older versions of packages which have alternatives in -updates and -security
+    # This will mess up a little with the system but all should be fine for the test purpose
+    And I apt install `<packages_to_install>`
```

Comment:
Nit: please separate this into two variables for the updates update and the security update.

Reply:
done